src/os/bluestore/BlueStore.cc

```diff
@@ -3543,6 +3538,36 @@ void BlueStore::_set_blob_size()
            << std::dec << dendl;
 }
 
+int BlueStore::_set_cache_sizes()
+{
+  cache_meta_ratio = cct->_conf->bluestore_cache_meta_ratio;
+  cache_kv_ratio = cct->_conf->bluestore_cache_kv_ratio;
+  cache_data_ratio = 1.0 - cache_meta_ratio - cache_kv_ratio;
+
+  if (cache_meta_ratio <= 0 || cache_meta_ratio > 1.0) {
```

Comment:
we error out in this case so the values aren't used.  this is mostly just less typing